plugins/rtmp-services/data/services.json

```diff
@@ -307,14 +307,10 @@
         {
             "name": "VIMM",
             "servers": [
-                {
-                    "name": "Europe: Frankfurt",
-                    "url": "rtmp://eu.vimm.tv/live"
-                },
-                {
-                    "name": "North America: Montreal",
-                    "url": "rtmp://us.vimm.tv/live"
-                }
+		{
+	            "name": "Autodetect",
+                    "url": "rtmp://ingest.vimm.tv/live"
+		}
```

Comment:
Mixed spaces / tabs here.

Reply:
cant fix that 1 extra space, it wont let me fix it.

Reply:
there i fixed that issue now it was bottom } issue

Reply:
Now you changed the formatting completely. Keep the indentation and style for your changes the same as the rest of the file and squash your commits afterwards to one single, correctly titled commit.

Make sure you read the [Contributing Guidelines](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
Also, this PR's current title prefix is not valid, see the history for this file.

Reply:
> Squash your commits to one single, correctly titled commit and make sure you read the [Contributing Guidelines](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
> Also, this PR's current title prefix is not valid, see the history for this file.

how i pose do that when i legit just pushed it out, sorry its bit annoying everyone but i never ever pushed this update before so pretty much need expect that im new pushing update service that i havent done before. and none this stuff wasnt told me start of it like title or bio info it needed. before on my screen was just blank.